### ISSUE TITLE
Allow adding custom fine-tuned whisper models

### ISSUE THREAD

**Issue by mathieuromain08** [kaixxx/noScribe#107]
Hello,

It would be great to be able to choose your own model, there are fine tune models for French, or other languages, for example that are much better than the basic Whisper ones. Perhaps to keep the “out of the box” aspect, three choices of models, fast, precise and custom?

Anyway, thanks for your work, absolutely incredible quality, I've been looking for this for a long time for my radio and interview work.

**Comment by kaixxx** [comment]
I totally agree. We plan to implement a mechanism to add extra models. Which one would you recommend for french in particular? It would be nice to have a curated list of good working models. 

Currently, you can try to add your custom model manually: 
- It must be in the 'faster-whisper' format: https://github.com/SYSTRAN/faster-whisper Only fine-tuned models based on whisper-large v2 or smaller models are supported right now (at least on Windows; on Mac, newer models might work too).
- In the installation-folder of noScribe, you will find the sub folder "models". You can replace the contents of either "faster-whisper-small" or "faster-whisper-large-v2" with your custom fine-tuned model. However, make sure that all the file and folder names are exactly the same.

**Comment by Lod3** [comment]
> It would be nice to have a curated list of good working models.
This might be a good excuse to setup a wiki part on this repo to have a matrix of good performing models per language.

**Comment by mathieuromain08** [comment]
Thanks, for the moment the model I was testing is not compatible, an error linked to a too recent version of CTranslate2. For your information, it's this model: https://huggingface.co/bofenghuang/whisper-large-v2-french

**Comment by kaixxx** [comment]
OK, thanks for testing. I will update CTranslate2 with the next release.
